library/proc_macro/src/lib.rs

```diff
@@ -51,11 +52,24 @@ use std::{error, fmt};
 
 #[unstable(feature = "proc_macro_diagnostic", issue = "54140")]
 pub use diagnostic::{Diagnostic, Level, MultiSpan};
+#[unstable(feature = "proc_macro_value", issue = "136652")]
+pub use rustc_literal_escaper::EscapeError;
```

Comment:
We might want to define our own error enum instead of using the `rustc_literal_escaper` one since any changes to it will directly impact the proc-macro crate.